README.adoc

```diff
@@ -170,11 +162,11 @@ include::finish/order/pom.xml[]
 
 In order to develop the MicroProfile Reactive Messaging application using Maven, add the [hotspot=reactiveMessaging file=2]`microprofile-reactive-messaging-api` and [hotspot=kafka file=2]`kafka-clients` dependencies to the [hotspot file=2]`pom.xml`
 
-=== Building the other microservices
+=== Completing the other microservices
 
-The `Kitchen`, and `Status` microservices are put together in this section.
+The `Bar`, `ServingWindow`, and `OpenLibertyCafe` microservices are provided to you. You need to complete the implementation of the `Kitchen` and `Status` microservices in this section.
```

Comment:
While this is good. The reason why I didn't include those details is because it makes the reader overwhelmed with so many micro services.

Reply:
updated